roadmap/index.html

```diff
@@ -36,7 +36,7 @@ <h1>Future Direction</h1>
 <section>
 <p>
 The future direction of GeoServer is primarily shaped by the constant flow of code contributions and funding directed towards the implementation of new features.
-In case you are wondering about a new feature or improvement we have a <a href="{{ site.wiki_url }}/Successfully-requesting-and-integrating-fixes,-improvements-and-new-features-in-GeoServer" >wiki page describing the process to discuss and implement it</a>.
+In case you are wondering about a new feature or improvement we have a <a href="{{ site.wiki_url }}/Successfully-requesting-and-integrating-new-features-and-improvements-in-GeoServer" >wiki page describing the process to discuss and implement it</a>.
```

Comment:
```suggestion
In case you are wondering about a new feature or improvement we have a <a href="{{ site.wiki_url }}/Successfully-requesting-and-integrating-fixes,-improvements-and-new-features-in-GeoServer,-GeoTools-and-GeoWebCache" >wiki page describing the process to discuss and implement it</a>.
```